index.html

```diff
@@ -30,52 +38,62 @@ <h1>Documentation</h1>
     <p>
       Password Purgatory is an intentionally infuriating API to request inane and ultimately unachievable password
       criteria intended to deliberately frustrate the user. Regardless of the password used, it will always be rejected
-      hence subjecting the user to "purgatory". Read more in <a
-        href="https://www.troyhunt.com/building-password-purgatory-with-cloudflare-pages-and-workers/">the launch blog
-        post</a>.
+      hence subjecting the user to "purgatory". Read more in
+      <a href="https://www.troyhunt.com/building-password-purgatory-with-cloudflare-pages-and-workers/">
+        the launch blog post</a>.
     </p>
-    <h2>Request</h2>
-    <p>
-      There is a single API endpoint that can be invoked with a GET request passing the password as a query string:
-    </p>
-    <code>GET https://api.passwordpurgatory.com/make-hell?password={password}</code>
-    <h2>Response</h2>
-    <p>
-      A JSON response is returned containing a message describing how the password did not meet the required criteria:
-    </p>
-    <code>
-    {
-      "message": "Password must contain at least 1 number"
-    }
-  </code>
-    <h2>Cross-Origin Resource Sharing (CORS)</h2>
-    <p>
-      CORS is enabled to allow calling the API from all origins.
-    </p>
-    <h2>Embedding in an External Website</h2>
-    <p>
-      To orchestrate the request to the API and display of the returned message, the "make-hell.js" JavaScript file can
-      be
-      directly embedded in any consuming websites:
-    </p>
-    <code>
-    &lt;script src="https://passwordpurgatory.com/make-hell.js"&gt;&lt;/script&gt;
-  </code>
-    <p>
-      ToDo: implement SRI integrity check once version is stable
-    </p>
-    <h2>Source Code</h2>
-    <p>
-      All code in Password Purgatory is open source and available on GitHub in the following repositories:
-    </p>
-    <ol>
-      <li>This website: <a
-          href="https://github.com/troyhunt/password-purgatory">github.com/troyhunt/password-purgatory</a></li>
-      <li>The API: <a
-          href="https://github.com/troyhunt/password-purgatory-api">github.com/troyhunt/password-purgatory-api</a></li>
-    </ol>
+
+    <article>
+      <h2>Request</h2>
+      <p>
+        There is a single API endpoint that can be invoked with a GET request passing the password as a query string:
+      </p>
+      <code>GET https://api.passwordpurgatory.com/make-hell?password={password}</code>
+    </article>
```

Comment:
This is not an article. From [the HTML spec](https://html.spec.whatwg.org/#the-article-element):

> The [article](https://html.spec.whatwg.org/#the-article-element) element [represents](https://html.spec.whatwg.org/#represents) a complete, or self-contained, composition in a document, page, application, or site and that is, in principle, independently distributable or reusable, e.g. in syndication. This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, an interactive widget or gadget, or any other independent item of content.

The contents of this tag is not a complete self contained composition, it is part of a larger composition that documents the API. If you pull this out of the page and put it on its own page, it would make no sense. Hence, it should not be in an article tag.